### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by zaxwebs**
This will be helpful in helping others better understand this API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,20 @@
-# api
+# COVID19-India API
 
-https://api.covid19india.org/data.json
+A  volunteer-driven API for COVID-19 stats & patient tracing in India (Unofficial).
 
-https://api.covid19india.org/state_district_wise.json
+## API
+* National time series, statewise stats and test counts : https://api.covid19india.org/data.json
```
Comment (satyarohith) [inline README.md]:
```suggestion
* National time series, statewise stats and test counts: https://api.covid19india.org/data.json
```

**Comment by ars75** [comment]
I would recommend making it as a Table, it looks much better for documentation:

## API
| Data  | URL   |
|-------|-------|
| National time series, statewise stats and test counts | https://api.covid19india.org/data.json |
| State-district-wise | https://api.covid19india.org/state_district_wise.json |
| Travel history| https://api.covid19india.org/travel_history.json |
| Raw data | https://api.covid19india.org/raw_data.json |

**Comment by zaxwebs** [comment]
Thank you for the suggestion @ars75. I've added it in.

**Comment by zaxwebs** [comment]
Is there a general direction you'd like me to take this in, in terms of tone, content and/or technical representation? @JunaidBabu @sudevschiz

**Pull request merged**